### PR TITLE
API cleanups, pull webrtcvad_js from npm.

### PR DESCRIPTION
**Pull request by mrstegeman**

### CODE REVIEW

**Comment by autonome** [comment]
Looking now, thanks for the poke.

File: test.html [hunk 1]
```diff
@@ -31,199 +31,197 @@ <h1>SpeakToMe - Speech Recognition API from Mozilla</h1>
...
-      test.components.unshift(['listen']);
-
-      // Make array of functions that each return the test component
-      // as a promise w/ test component parameters applied to it.
-      var promises = test.components.map((c) => {
-        return () => {
-          return testcomponents[c[0]].apply(testcomponents[c[0]], [test].concat(c[1]));
-        };
+    */
+    function validateResults(stmResults, expectedResults) {
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply test.html]: Apparently, this file used tabs. Fixed.

File: src/stm_vad.js [hunk 2]
```diff
@@ -22,31 +22,33 @@ function SpeakToMeVAD(options) {
...
   //
   var buffer_vad = new Int16Array(sizeBufferVad);
   //
   var leftovers = 0;
   //
-  var finishedVoice = false;
-  //
   var samplesvoice = 0;
   //
+  var samplessilence = 0;
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply src/stm_vad.js]: `samplessilence` was used but was never actually defined. `finishedVoice` was never actually used.

File: test.html [hunk 3]
```diff
@@ -31,199 +31,197 @@ <h1>SpeakToMe - Speech Recognition API from Mozilla</h1>
...
-  //runTest(tests[1])
-  //runTests(tests)
-
-});
+
+    // Tests
+    var tests = [{
+      name: 'voice "test"',
+      components: [
+	['voice', ['test']]
```
[the remark this answers is not in the document]
Comment (mrstegeman) [reply test.html]: Fixed.

**Pull request merged**